src/test/Makefile

```diff
@@ -515,8 +515,8 @@ GTEST_DIR = ../../lib/test/gtest
 
 # Use clang/clang++ by default
 
-CC  := clang-15
-CXX := clang++-15
+CC  := clang-18
+CXX := clang++-18
 ifeq ($(shell which $(CC) 2>/dev/null),)
```

Comment:
Can there be another check here? For example I have clang-18 but for some reason I don't have clang++-18 so I get this error
```console
make[1]: clang++-18: No such file or directory
make[1]: *** [Makefile:814: ../../obj/test/alignsensor_unittest/alignsensor_unittest.o] Error 127
```
the fix would be 
```makefile
CC  := clang-18
CXX := clang++-18
ifeq ($(shell which $(CC) 2>/dev/null),)
$(info Falling back to 'clang')
CC  := clang
endif
ifeq ($(shell which $(CXX) 2>/dev/null),)
$(info Falling back to 'clang++')
CXX := clang++
endif
```

Reply:
the suggestion may be problematic. It may end up mixing different c and cxx.
test both, fallback if any is missing.

Reply:
Consolidated.